docs/Users_Guide/existing_builds_template.rst

```diff
@@ -0,0 +1,587 @@
+
+              * /ncrc/proj/nggps_psd/user_name/projects/METplus/sample_data/METplus-X.Y_sample_data
+
+            * To use METplus run: Users should create a file like 
+              /gpfs/f5/esrl/proj-shared/user_name/projects/METplus/gaea.metplus.conf 
+              to set a personalized INPUT_BASE and OUTPUT_BASE.
+
+              .. code-block:: ini
+
+                 module unload cray-libsci/23.12.1.1
+                 module load intel-oneapi/2022.0.2
+                 module use /usw/met/METplus/modulefiles
+                 module load metplus/X.Y.0-beta1
+
+          * **MET-X.Y.0-beta1**
+
+            * MODULES:
+
+              .. code-block:: ini
+
+                 module unload cray-libsci/23.12.1.1
+                 module load intel-oneapi/2022.0.2
+                 module use -a /usw/met/modulefiles/
+                 module load met/X.Y.0-beta1
+
+
+          * **METcalcpy-X.Y.0-beta1 / METplotpy-X.Y.0-beta1**
+
+            * MODULES:
+
+              .. code-block:: ini
+
+                module unload cray-libsci/23.12.1.1
+                module load intel-oneapi/2022.0.2
+                module use /usw/met/METcalcpy/modulefiles
+                module load metcalcpy/X.Y.0-beta1
+                module use /usw/met/METplotpy/modulefiles
+                module load metplotpy/X.Y.0-beta1
+
+          * **METdataio-X.Y.0-beta1**
+
+            * MODULES:
+
+              .. code-block:: ini
+
+                 module unload cray-libsci/23.12.1.1
+                 module load intel-oneapi/2022.0.2
+                 module use /usw/met/METdataio/modulefiles
+                 module load metdataio/X.Y.0-beta1
+
+.. dropdown:: Community machines - Coming Soon!
+
+     .. dropdown:: FRONTERA - Coming Soon!
+
+        | **TEXAS ADVANCED COMPUTING CENTER (TACC) FRONTERA**
+        | *Last Updated:*
+
+          * **MET-X.Y.0-beta1**
+
+            * MODULES: 
+
+          * **METplus-X.Y.0-beta1**
+
+            * METplus-X.Y.0-beta1 Installation
+            * METplus-X.Y Sample Data
+            * To set up the environment run:
+            * Users should create a file like /work2/06612/tg859120/frontera/METplus/frontera.user_name.conf
+              to set a personalized INPUT_BASE and OUTPUT_BASE.
+
+.. dropdown:: DockerHub - Coming Soon!
+
+   | **MET**
+   | *Last Updated:*
+
+      .. code-block:: ini
+
+          docker pull dtcenter/met:X.Y.0-beta1
+
+     `dtcenter/met DockerHub <https://hub.docker.com/r/dtcenter/met>`_
+
+   | **METplus**
+   | *Last Updated:*
+
+      .. code-block:: ini
+
+          docker pull dtcenter/metplus:X.Y.0-beta1
+
+     `dtcenter/metplus DockerHub <https://hub.docker.com/r/dtcenter/metplus>`_
+
+   | **METplus Analysis**
+   | *Last Updated:*
+
+      .. code-block:: ini
+
+          docker pull dtcenter/metplus-analysis:X.Y.0-beta1
+
+     `dtcenter/metplus-analysis DockerHub <https://hub.docker.com/r/dtcenter/metplus-analysis>`_
+
+.. dropdown:: AWS - Coming Soon!
```

Comment:
Wondering if this section will continue to be needed since the NOAA/EMC AWS METviewer instance is being decomissioned. Not a big deal though.